api/Dockerfile

```diff
@@ -1,14 +1,39 @@
-FROM node:18.15.0-alpine
+##################
+### BASE IMAGE ###
+##################
+FROM node:18.15.0-alpine AS base
+
+# Directory used in container
+WORKDIR /usr/api/
+
+# Copy dependency package, install, copy everything else
+COPY package*.json .
+RUN npm i --omit=dev
```

Comment:
You only need typescript installed to run the build command, then you can install packages in the prod step

Reply:
Noted. Working without the packages.